js/app.js

```diff
@@ -1,10 +1,31 @@
 
 if (localCheck){
 
-  
- 
+  // if we have local storage, read from anything we saved from before
+  // const items = JSON.parse(localStorage.getItem('items'));
+  // if (items) {
+  //   console.log("Items: " + items);
+  // } else {
+  //   console.log('No items! Setting test items')
+  //   setItems(items);
+  // };
+
 } 
 
+
+function setItems() {
+  console.log('in setItems');
+
+  const testItem = {
+    'test': 'item'
+  }
+
+  localStorage.setItem('items', JSON.stringify(testItem));
+
+  const newItems = JSON.parse(localStorage.getItem('items'));
+  console.log("Items from storage:");
+  console.log(newItems);
+};
```

Comment:
This and above can be ignored, just the below stuff is what's being used for now